components/digital_ocean/actions/add-ssh-key/add-ssh-key.mjs

```diff
@@ -1,45 +1,32 @@
-// legacy_hash_id: a_8KiVPz
-import doWrapperModule from "do-wrapper";
+import digitalOceanApp from "../../digital_ocean.app.mjs";
 
 export default {
   key: "digital_ocean-add-ssh-key",
   name: "Add SSH Key",
   description: "Adds a new SSH to your account.",
```

Comment:
All actions should include a link to the relevant documentation. This applies to all of the component actions.
```suggestion
  description: "Adds a new SSH to your account. [See the docs here](https://docs.digitalocean.com/reference/api/api-reference/#operation/create_ssh_key)",
```